.github/CONTRIBUTOR_GUIDELINES.md

```diff
@@ -0,0 +1,110 @@
+# Contributing to OpenCollar
+
+We welcome contributions to OpenCollar of any kind including documentation,
+organization,bug reports, issues, feature requests, feature implementations,
+pull requests, answering questions in the group / discord, helping to manage issues, etc.
+
```

Comment:
Replace 3-5 and add here: " This document lists ways you can contribute to developing the OpenCollar source code. Listed here are:  Reporting issues such as bug reports and feature requests, submitting pull requests, and reviewing pull requests.  Other ways you may contribute (not covered here):  Inworld help, asking and answering questions in the group/discord, documentation, checking vendor locations and reporting if they are out of date, beta testing, and building your collars with our scripts.  We welcome any and all of these."